component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java

```diff
@@ -577,8 +578,9 @@ public boolean isActivityDeletable() throws SpaceException {
       }
     }
 
+    UserACL userACL = getApplicationComponent(UserACL.class);
     if (space != null) {
-      return spaceService.isManager(space, Utils.getOwnerRemoteId());
+      return userACL.isSuperUser() || spaceService.isManager(space, Utils.getOwnerRemoteId()) || ConversationState.getCurrent().getIdentity().getGroups().contains(userACL.getAdminGroups());
```

Comment:
We need to check if the user is a super manager (member of space administration group)
```suggestion
      return userACL.isSuperUser() || spaceService.isManager(space, Utils.getOwnerRemoteId()) || spaceService.isSuperManager(Utils.getOwnerRemoteId())
```

Reply:
Okay i will add it